scalastyle-config.xml

```diff
@@ -0,0 +1,93 @@
+<scalastyle>
```

Comment:
I don't know how everyone feels about scalastyle but I don't think it adds too much value and ends up being more of a pain than a help. Definitely just my opinion but I'd get rid.